### PR TITLE
Add Overwrite Warning to Moment Map Plugin

### PR DESCRIPTION
**Pull request by duytnguyendtn**
### Description

This PR adds a visual prompt to the user when overwriting an existing file on disk. It allows the user to cancel or continue and overwrite the file.

I believe there are multiple places where we write digital artifacts for the user; this PR can be the basis for a reusable mixin later down the line

https://user-images.githubusercontent.com/25206008/192894471-2c1c755c-732b-4f5f-b13f-3db3ab021975.mp4

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Review by kecnry — APPROVED** [review]
Nice and clean implementation - let's get this in!

**Review by rosteen — APPROVED** [review]
Nice solution, looks good on dark mode 😁

File: CHANGES.rst [hunk 1]
```diff
@@ -75,6 +75,9 @@ Cubeviz
 - Calling ``cubeviz.load_data(data, data_label)``, where ``data_label`` is passed in
   as second positional argument instead of keyword, is now allowed. [#1644]
 
+- A warning will be presented with a warnning when overwriting a moment map on
```
Comment (pllim) [inline CHANGES.rst:78]:
```suggestion
- A warning will be presented with a warning when overwriting a moment map on 
```

Comment (duytnguyendtn) [reply CHANGES.rst:78]: Hehe whoops, I missed that...

**Pull request merged**